pkg/monitoring/metrics/virt-controller/vmstats_collector_test.go

```diff
@@ -162,6 +162,44 @@ var _ = Describe("VM Stats Collector", func() {
 			}
 		})
 
+		It("should show machine_type correctly", func() {
+			vms := []*k6tv1.VirtualMachine{
+				{
+					Spec: k6tv1.VirtualMachineSpec{
+						Template: &k6tv1.VirtualMachineInstanceTemplateSpec{
+							Spec: k6tv1.VirtualMachineInstanceSpec{
+								Domain: k6tv1.DomainSpec{
+									Machine: &k6tv1.Machine{
+										Type: "q35",
+									},
+								},
+							},
+						},
+					},
+					Status: k6tv1.VirtualMachineStatus{
+						PrintableStatus: k6tv1.VirtualMachineStatusCrashLoopBackOff,
+					},
+				},
+			}
+
+			crs := CollectVMsInfo(vms)
+			Expect(crs).To(HaveLen(1))
+
+			for i, cr := range crs {
+				Expect(cr).ToNot(BeNil())
+				Expect(cr.Metric.GetOpts().Name).To(ContainSubstring("kubevirt_vm_info"))
+				Expect(cr.Value).To(BeEquivalentTo(1))
+				Expect(cr.Labels).To(HaveLen(10))
+
+				Expect(cr.GetLabelValue("name")).To(Equal(vms[i].ObjectMeta.Name))
+				Expect(cr.GetLabelValue("namespace")).To(Equal(vms[i].ObjectMeta.Namespace))
+				Expect(cr.GetLabelValue("machine_type")).To(Equal(vms[i].Spec.Template.Spec.Domain.Machine.Type))
+
+				Expect(cr.GetLabelValue("status")).To(Equal("CrashLoopBackOff"))
+				Expect(cr.GetLabelValue("status_group")).To(Equal("error"))
```

Comment:
this is duplicate code, why not add the machine type here: https://github.com/kubevirt/kubevirt/pull/13045/files#diff-5381e9e699430fdce5f8584e775dceb9721c103755e6482b56db37cfd2ec53eeR125 and add the `Expect(cr.GetLabelValue("machine_type")).To(Equal(vms[i].Spec.Template.Spec.Domain.Machine.Type))` here https://github.com/kubevirt/kubevirt/pull/13045/files#diff-5381e9e699430fdce5f8584e775dceb9721c103755e6482b56db37cfd2ec53eeR154?

Reply:
I sere it also in other tests so maybe I'm wrong here. @machadovilaca WDYT?

Reply:
i would be fine with that, just remember to tweak the test name

Reply:
sure